pkg/autodiscovery/configresolver_test.go

```diff
@@ -357,6 +357,25 @@ func TestResolve(t *testing.T) {
 			},
 			errorString: "envvar name is missing, skipping service a5901276aed1",
 		},
+		//// hostname
+		{
+			testName: "simple %%hostname%%",
+			svc: &dummyService{
+				ID:            "a5901276aed1",
+				ADIdentifiers: []string{"redis"},
+				Hostname:      "i.m.here",
```

Comment:
In fact this is not a hostname, this is a FQDN.
Could we use an another term for that to be more accurate about what it is?

Reply:
I'm not sure about that, I don't have the all picture here. Can you give me you opinion about that?

Reply:
Indeed, let's only get the hostname for now. If needed, we could leverage template vars to add a `%%hostname_full%%` later on